site/public/js/server.js

```diff
@@ -1,31 +1,32 @@
 ////////////Begin: Removed redundant link in breadcrumbs////////////////////////
 //See this pr for why we might want to remove this code at some point
+
```

Comment:
Most of the whitespace changes in this file are distracting, and unrelated to this PR.  It would be good to lint this file eventually, but such changes should happen in a different PR.